File: demo_call_api/src/main/java/org/fai/study/demo_call_api/controller/entities/Order.java
package org.fai.study.demo_call_api.controller.entities;

public class Order {
    private String id;
    private String customerId;
    private String customerName;
    private String customerPhone;
    private String customerAddress;

    public Order() {
    }

    public Order(String customerAddress, String customerId, String customerName, String customerPhone, String id) {
        this.customerAddress = customerAddress;
        this.customerId = customerId;
        this.customerName = customerName;
        this.customerPhone = customerPhone;
        this.id = id;
    }

    public String getCustomerAddress() {
        return customerAddress;
    }

    public void setCustomerAddress(String customerAddress) {
        this.customerAddress = customerAddress;
    }

    public String getCustomerId() {
        return customerId;
    }

    public void setCustomerId(String customerId) {
        this.customerId = customerId;
    }

    public String getCustomerName() {
        return customerName;
    }

    public void setCustomerName(String customerName) {
        this.customerName = customerName;
    }

    public String getCustomerPhone() {
        return customerPhone;
    }

    public void setCustomerPhone(String customerPhone) {
        this.customerPhone = customerPhone;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }
}

File: flutter_demo_callapi/lib/models/Order.dart
class Order {
  late String id;
  late String customerId;
  late String customerName;
  late String customerPhone;
  late String customerAddress;
  Order(this.id, this.customerId, this.customerName, this.customerPhone,
      this.customerAddress);
  factory Order.fromJson(Map<String, dynamic> json) {
    return Order(json['id'], json['customerId'], json['customerName'],
        json['customerPhone'], json["customerAddress"]);
  }
}
